tests/conftest.py

```diff
@@ -21,11 +24,19 @@
 from iam.models import db as db_
 
 
-@pytest.fixture(scope='session')
+@pytest.fixture(scope="session")
 def app():
+    """Provide an initialized Flask for use in certain test cases."""
     init_app(app_, api, db_)
-    app_.app_context().push()
-    return app_
+    with app_.app_context():
```

Comment:
If `test_mode` passes with this and the `app.config` is accessible, I'm also fine with changing this on the cookiecutter.

Reply:
It does, and furthermore I was not able to make all tests pass without it. I.e., writes to DB session would be visible in tests but not in the tested application, for a reason I don't yet understand, but might be related to the `flask-sqlalchemy` session handling.